presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/SemiTransactionalHiveMetastore.java

```diff
@@ -1007,7 +1008,7 @@ private void commitShared()
                     List<String> partitionValues = partitionEntry.getKey();
                     Action<PartitionAndMore> action = partitionEntry.getValue();
                     HdfsContext context = action.getContext();
-                    MetastoreContext metastoreContext = new MetastoreContext(action.getContext().getIdentity(), action.getContext().getQueryId().orElse(""), action.getContext().getClientInfo(), action.getContext().getSource());
+                    MetastoreContext metastoreContext = new MetastoreContext(action.getContext().getIdentity(), action.getContext().getQueryId().orElse(""), action.getContext().getClientInfo(), action.getContext().getSource(), action.getContext().getSession().flatMap(session -> getMetastoreHeaders(session)));
```

Comment:
`action.getContext().getSession().flatMap(MetastoreUtil::getMetastoreHeaders)`

Reply:
✅  Refactored other places as well.

Reply:
you missed this.

Reply:
Now it is showing the message, I think github is sometimes late on their notifications.